extensions/functions_boolean.yaml

```diff
@@ -138,3 +138,42 @@ aggregate_functions:
             name: a
         nullability: DECLARED_OUTPUT
         return: boolean?
+
+  -
+    name: "bit_or"
+    description: >
+      Calculates the bitwise OR of all values in set of values.
+      Null values doesn't affect the output and can be considered ignored.
+      If the output is null it means that there is no output for the passed input.
+    impls:
+      - args:
+          - name: x
+            value: boolean
+        nullability: DECLARED_OUTPUT
+        return: boolean
```

Comment:
I believe `DECLARED_OUTPUT` with a return type of `boolean` means that there cannot be nulls in the output.  Yet, the description claims "If the output is null it means there is no output for the passed input."  Do you perhaps want `return: boolean?`

Reply:
I agree. We should do that.